bedrock/products/templates/products/vpn/landing-refresh.html

```diff
@@ -83,7 +94,15 @@
     media_class='mzp-l-split-h-center',
     media_after=True
   ) %}
+  {% if entrypoint_experiment == 'vpn-headlines' and entrypoint_variation == '2' %}
+    <h1 class="u-title-xl">{{ ftl('vpn-landing-powerful-privacy-for-peace') }}</h1>
+    <h2 class="c-page-subtitle u-title-xs">{{ headline_exp_v2_sub }}</h2>
+  {% elif entrypoint_experiment == 'vpn-headlines' and entrypoint_variation == '3' %}
+    <h1>Mozilla VPN</h1>
+    <h2 class="c-page-subtitle u-title-xs">{{ headline_exp_v3_sub }}</h2>
```

Comment:
**Suggestion(design/non-blocking):** I think based on how we have most of our other pages laid out, we don't usually have our subheadings bolded (granted, they're usually `p` tags), so I'm thinking keeping the font-weight normal here would be more consistent with some of our other pages:

<kbd>![Screenshot 2024-02-07 at 11 57 31 AM](https://github.com/mozilla/bedrock/assets/42309026/dbd77747-9095-4339-9208-b16cbddc23f8)</kbd>

This is up to you to implement as it's at your discretion, but thought it'd be a good suggestion to throw in 👍🏼 plus I think it's a nice separator from the page title

Reply:
Good suggestion. Done.